app/validators/array_hostnames_validator.rb

```diff
@@ -1,5 +0,0 @@
-class ArrayHostnamesValidator < ActiveModel::EachValidator
```

Comment:
How about we keep this + the test so plugins can use this?

Reply:
Hai! Rebased.